apps/component-generator/README.md

```diff
@@ -4,14 +4,14 @@ It's used to automate process of component creation.
 
 ## Setup
 
-1. Run in your command prompt `npm i -g gulp-cli`
-2. Run `npm i` inside the component-generator directory
+1. Run in your command prompt `npm i -g gulp-cli`. May need to run this with `sudo`
```

Comment:
is "sudo" only a mac thing? lol i don't know how windows does admin permission.  run as admin terminal window?

Reply:
Yep. You have to open command prompt as an administrator, which you can do from it's context menu.

Reply:
For me , it ran without admin permissions on windows.

Reply:
I'll add a comment specifying that the `sudo` part is only for mac!